components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/provisioning/impl/DefaultProvisioningHandler.java

```diff
@@ -81,6 +81,7 @@
     private static final Log log = LogFactory.getLog(DefaultProvisioningHandler.class);
     private static final String USER_WORKFLOW_ENGAGED_ERROR_CODE = "WFM-10001";
     private static volatile DefaultProvisioningHandler instance;
+    private static final String LOCAL_DEFAULT_CLAIM_DIALECT = "http://wso2.org/claims";
```

Comment:
Don't we've a public constant defined for this in framework?

Reply:
Currently we have a public constant with name `LOCAL_IDP_DEFAULT_CLAIM_DIALECT` that has the same value, but using it here is miss leading with the name.